content/create-an-addon/react/en/decorators.md

```diff
@@ -75,7 +75,7 @@ export const withGlobals = (StoryFn, context) => {
   }, [context.id]);
 
   useEffect(() => {
-    const selectorId = isInDocs ? `addon-outline-docs-${context.id}` : `addon-outline`;
+    const selectorId = isInDocs ? `addon-outline-docs-${context.id}` : `my-addon-outline`;
```

Comment:
You also want the `my-` prefix in front of the docs selector otherwise it still won't work in the docs panel.
```suggestion
    const selectorId = isInDocs ? `my-addon-outline-docs-${context.id}` : `my-addon-outline`;
```